### DISCUSSION TITLE
Add visual scrollbar to beatmap cards with over 10 difficulties

### DISCUSSION THREAD

**Discussion by zachmanthethird** [ppy/osu#16514] [Ideas] [answered]
Builds on #16307 and #16120.
It isn't intuitively obvious that you can scroll through the difficulties. I thought it was a padding issue until I actually looked at #16307 to find the implemented solution. Version 2022.118.0
![osu_2022-01-18_12-48-52](https://user-images.githubusercontent.com/74262407/150016996-36439f47-2239-413f-8a53-b01ed88f0b75.png)
Proposed solution:
![image](https://user-images.githubusercontent.com/74262407/150017737-5c476b97-6122-418b-8ec8-0d251d4930a8.png)

**Comment by bdach** [comment]
i could have sworn web didn't use to have a scrollbar there but now does...??

either way yeah, probably needed.

**Comment by peppy** [comment]
It also looks like there's no enough padding here. The bottom of the card should be visible.

**Comment by bdach** [comment, marked as answer]
Both points addressed in https://github.com/ppy/osu/pull/16525
